src/_includes/layouts/resources.njk

```diff
@@ -0,0 +1,58 @@
+{% if previewMode %}
+    {% extends 'layouts/preview.njk' %}
+{% else %}
+    {% extends 'layouts/base.njk' %}
+{% endif %}
+
+{% set pageType = 'page--resources' %}
+
+{% block content %}
+    <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+        <symbol id="previous" viewBox="0 0 26 26" fill="none">
+        {% include 'svg/previous.svg' %}
+        </symbol>
+        <symbol id="next" viewBox="0 0 26 26" fill="none">
+        {% include 'svg/next.svg' %}
+        </symbol>
+        <symbol id="topic" viewBox="0 0 14 20" fill="none">
+        {% include 'svg/icon-topics.svg' %}
+        </symbol>
+        <symbol id="type" viewBox="0 0 16 18" fill="none">
+        {% include 'svg/icon-resource-type.svg' %}
+        </symbol>
+        <symbol id="plus" width="10" height="10" viewBox="0 0 10 10" fill="none">
+        {% include 'svg/icon-plus.svg' %}
+        </symbol>
+        <symbol id="minus" width="10" height="2" viewBox="0 0 10 2" fill="none">
+        {% include 'svg/icon-minus.svg' %}
+        </symbol>
+        <symbol id="external" width="16" height="16" viewBox="0 0 16 16" fill="none">
+        {% include 'svg/external.svg' %}
+        </symbol>
+        <symbol id="close" width="20" height="20" viewBox="0 0 20 20" fill="none">
+        {% include 'svg/icon-x.svg' %}
+        </symbol>
+    </svg>
+    
+    <article>
+        {% include 'partials/components/page-header.njk' %}
+        <div class="resources-container">
+            <form method="get" action="/resources/">
+                {% include 'partials/components/filter-resources.njk' %}
+            </form>
+
+            <div class="content">
+                {# content of this block is generated from renderSearchResults and renderResources methods in resources-utils, #}
```

Comment:
This comment can be improved to something like `This container lists either all resources or resources filtered based on user's filtering selections. It's dynamically rendered at the page load by `resources-dynamic-handler.js`". Feel free to improve further.